airbyte-api/src/main/openapi/config.yaml

```diff
@@ -1402,6 +1402,29 @@ paths:
           $ref: "#/components/responses/NotFoundResponse"
         "422":
           $ref: "#/components/responses/InvalidInputResponse"
+  /v1/state/update:
```

Comment:
I think we shouldn't have this endpoint. We have a "contract" with the connector which is that the platform never modify the content of a state, it will be only done by the connector. I think that the user will have less context of what is a state than the platform and shouldn't be allowed to modify it. It can lead to a lot of issue like data duplication or data missing.

From where that need come from?

Reply:
Looks like this is blowing up more than we initially expected!  I'll add more of the rationale and link back to the user issues up in the description.

Reply:
I think that we shouldn't make the modification of the state easy.  For technical support and power users, they can update it in the DB directly. I understand the need to display it for clarity purpose but the update has many potential side effect that the user might not fully understand.